### PR TITLE
Webc language support

### PR DESCRIPTION
**Pull request by vanarok**
Add [webc](https://github.com/11ty/webc) grammar.

### CODE REVIEW

File: runtime/queries/webc/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+(tag_name) @tag
+(erroneous_end_tag_name) @tag.error
```
Comment (the-mikedavis) [inline runtime/queries/webc/highlights.scm:2]:
```suggestion
(erroneous_end_tag_name) @error
```

File: runtime/queries/webc/highlights.scm
Comment (the-mikedavis) [inline runtime/queries/webc/highlights.scm:1]: Are these copies of the ones in `runtime/queries/html`? If so you can include them without duplicating the queries with

```
; inherits: html
```

Comment (vanarok) [reply runtime/queries/webc/highlights.scm:1]: Yep, it is.

**Comment by the-mikedavis** [comment]
This will need a run of `cargo xtask docgen` to fix the CI

**Review by pascalkuthe — APPROVED** [review]

**Comment by johannesrave** [comment]
very cool! is there a way for me to use this now without waiting for the next release if i can't rabbithole into getting the buildchain up atm?

**Comment by johannesrave** [comment]
thanks for working on this @vanarok ! i have by now figured out how to compile helix from master, and now the new version is released for packaging too.

i have another question regarding the actual grammar though:
auto-formatting breaks indented frontmatter at the moment, for example with indented arrays:

```
---
layout: "_layouts/document.webc"
title: recent posts
pagination:
# these lines get un-indented on :fmt
  data: collections.post 
  size: 2
  alias: posts
  reverse: true
---
<p>welcome to the rest of the page.</p>
```
looking at it with `:tree-sitter-subtree` the frontmatter is simply parsed as a string by the html-grammar.

i expected that the `injections.scm` in `queries/webc` somehow extends the `html` grammar-injections that `webc` uses as defined in the global `languages.toml`, and that i would be able to copy the yaml injections from the markdown-grammar:

```
; inherits: html
((minus_metadata) @injection.content (#set! injection.language "yaml") (#set! injection.include-unnamed-children))
```

 however, i see no effect after recompiling or rebuilding the grammars on their own. is `queries/webc` even used if the `languages.toml` defines the grammar for `webc` to be `html`?  

i tried working on this but have been unsuccessful, i just don't know enough about the whole build-chain.
i don't fully understand the interplay of the tree-sitter parsers that get pulled on build, the language-definitions and the `queries`-directories (with injections.scm).

**Pull request merged**